astroquery/mast/cutouts.py

```diff
@@ -314,3 +314,228 @@ def get_cutouts(self, coordinates=None, size=5, sector=None, objectname=None):
+            One and only one of coordinates and objectname must be supplied.
+        radius : str, float, or `~astropy.units.Quantity` object, optional
+            Default 0.2 degrees.
+            Accepted units are: d (degrees), s (arcseconds), m (arcminutes), px (pixels)
+            If supplied as a float, degrees is the assumed unit.
+            The string must be parsable by `~astropy.coordinates.Angle`. The
+            appropriate `~astropy.units.Quantity` object from
+            `astropy.units` may also be used.
+
+        Returns
+        -------
+        response :
+        Surveys for given coordinates
+        """
+
+        # Get Skycoord object for coordinates/object
+        coordinates = parse_input_location(coordinates)
+
+        params = {"ra": coordinates.ra.deg,
+                  "dec": coordinates.dec.deg,
+                  "radius": radius}
+
+        # If radius is just a number we assume degrees
+        if isinstance(radius, float):
+            radius = Angle(radius, u.deg)
+            params["radius"] = radius.deg
+
+        response = self._service_api_connection.service_request_async("survey", params)
+        response.raise_for_status()  # Raise any errors
+
+        survey_json = response.json()['surveys']
+
+        if not len(survey_json):
+            warnings.warn("Coordinates are not in any Z survey.", NoResultsWarning)
+        return survey_json
+
+    def download_cutouts(self, coordinates, size=5, units="px", survey=None, form="fits", path=".", **img_params):
+        """
+        Download cutout target pixel file(s) around the given coordinates with indicated size.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates` object
+            The target around which to search. It may be specified as a
+            string or as the appropriate `astropy.coordinates` object.
+            One and only one of coordinates and objectname must be supplied.
+        size : int, array-like, `~astropy.units.Quantity`
+            Optional, default 5 pixels.
+            The size of the cutout array. If ``size`` is a scalar number or
+            a scalar `~astropy.units.Quantity`, then a square cutout of ``size``
+            will be created.  If ``size`` has two elements, they should be in
+            ``(ny, nx)`` order.  Scalar numbers in ``size`` are assumed to be in
+            units of pixels. `~astropy.units.Quantity` objects must be in pixel or
+            angular units.
+        units : str
+            Optional, default is px.
+            The `units` parameter accepts the following units: d (degrees),
+            s (arcseconds), m (arcminutes), and px (pixels).
+        survey : str
+            Optional
+            The survey to restrict the cutout. The survey parameter will restrict to
+            only the matching survey. Default behavior is to return all matched surveys.
+        form : str
```

Comment:
I assume you used "form" instead of "format" because format already means something in Python, but let's change it to cutout_format. I know it's longer, but I think clarity is more important than conciseness here.